api/src/main/java/uk/gov/dhsc/htbhf/claimant/ApiDocumentation.java

```diff
@@ -62,6 +68,11 @@ private ApiInfo getApiInfo() {
 
     @EventListener(ApplicationReadyEvent.class)
     public void logAfterStartup() {
-        log.info("Application started. App version={}, app id={}, instance index={}", appVersion, applicationId, instanceIndex);
+        eventLogger.logEvent(ApplicationStartedEvent.builder()
```

Comment:
I know this isn't specifically done with this PR, but it feels like this method is in the wrong place, could we move it to `ClaimantServiceApplication`, I'd expect it to be somewhere more central like that rather than being associated with the Swagger doc generation.

Reply:
BTW, this is looking really nice now! :-)

Reply:
agreed, it wasn't really in the right place